app/src/main/java/com/kks/nimblesurveyjetpackcompose/ui/presentation/common/HomeScreenShimmerLoading.kt

```diff
@@ -0,0 +1,134 @@
+package com.kks.nimblesurveyjetpackcompose.ui.presentation.common
+
+import android.content.res.Configuration.UI_MODE_NIGHT_YES
+import androidx.compose.animation.core.*
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.*
+import androidx.compose.foundation.shape.CircleShape
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.clip
+import androidx.compose.ui.geometry.Offset
+import androidx.compose.ui.graphics.Brush
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.Dp
+import androidx.compose.ui.unit.dp
+import androidx.constraintlayout.compose.ConstraintLayout
+
+@Composable
+fun HomeScreenShimmerLoading() {
```

Comment:
Why don't we create a composable that returns a `Brush` called `shimmerBrush()` or something similar? Then we could reuse it on both the `ShimmerHomeScreen` and the previews, what do you think?
```suggestion
fun shimmerBrush(color: Color): Brush {
```